.gitlab-ci.yml

```diff
@@ -416,6 +416,16 @@ tcid-iuod12-volume-upgrade-check:
     - chmod 755 ./stages/10-openebs-upgrade-check/volume-upgrade-check
     - ./stages/10-openebs-upgrade-check/volume-upgrade-check
 
+## Volume upgrade Jiva Check
+tcid-iuod27-volume-upgrade-jiva-check:
+  image: harshshekhar15/gitlab-job:v1
+  stage: OPENEBS-UPGRADE-CHECK
+  dependencies:
+    - cluster-setup-check
+  script:
```

Comment:
is it required to have 10 as prefix?
How about below?
```yaml
  script:
    - chmod 755 ./stages/iuod27-openebs-upgrade-check/run
    - ./stages/iuod27-openebs-upgrade-check/run
```

cc @ajeshbaby @harshita-sharma011 @harshshekhar15